src/tuning-module.ts

```diff
@@ -1,72 +1,73 @@
-namespace tuning {
+import * as d3 from "d3"
+import * as events from "./events-module"
+import * as music from "./music-module"
 
-    let guitarDots: Array<[number, number]> = [
-        [3, 0], // [fret, position]
-        [5, 0],
-        [7, 0],
-        [9, 0],
-        [12, -1],
-        [12, 1],
-        [15, 0]
-    ];
+const guitarDots: Array<[number, number]> = [
+    [3, 0], // [fret, position]
+    [5, 0],
+    [7, 0],
+    [9, 0],
+    [12, -1],
+    [12, 1],
+    [15, 0],
+]
 
-    interface TuningInfo {
-        readonly tuning: string;
-        readonly dots: Array<[number, number]>;
-        readonly description: string;
-    }
+interface TuningInfo {
+    readonly tuning: string
+    readonly dots: Array<[number, number]>
+    readonly description: string
+}
 
-    let tunings: Array<TuningInfo> = [
-        { tuning: "EADGBE", dots: guitarDots, description: "Guitar Standard" },
-        { tuning: "EADGCF", dots: guitarDots, description: "All Fourths" },
-        { tuning: "CGDAEB", dots: guitarDots, description: "All Fifths" },
-        { tuning: "DADGBE", dots: guitarDots, description: "Guitar Drop D" },
-        { tuning: "DADGAD", dots: guitarDots, description: "Celtic Tuning" },
-        { tuning: "CGDAEA", dots: guitarDots, description: "Guitar Fripp NST" },
-        { tuning: "EADG", dots: guitarDots, description: "Bass Standard" },
-        { tuning: "DADG", dots: guitarDots, description: "Bass Drop D" },
-        { tuning: "DGBD", dots: guitarDots, description: "Banjo" },
-        { tuning: "DGBD", dots: guitarDots, description: "Cavaquinho"},
-        { tuning: "GCEA", dots: guitarDots, description: "Ukulele C" },
-        { tuning: "CGDA", dots: guitarDots, description: "Cello" },
-        { tuning: "GDAE", dots: guitarDots, description: "Violin" },
-        { tuning: "CGDA", dots: guitarDots, description: "Viola" },
-    ]
+const tunings: TuningInfo[] = [
```

Comment:
Another example of an automatic TSLint `--fix` change: `TuningInfo[]` is preferred over  `Array<TuningInfo>`.